.github/workflows/http_parser.yaml

```diff
@@ -0,0 +1,71 @@
+name: package:http_parser
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Why on both `master` and `main`? This repo is still using `master` (though, after all the dust has settled, we should switch it to `main`).

Reply:
This was purely cargo culting from the existing workflows in this repo.